main/inc/lib/myspace.lib.php

```diff
@@ -1156,137 +1156,253 @@ public static function displayResumeLP(
     ) {
         $tableHtml = '';
         $tblExtraField = Database::get_main_table(TABLE_EXTRA_FIELD);
-        $tblExtraFieldValue = Database::get_main_table(TABLE_EXTRA_FIELD_VALUES);
         $tblCourse = Database::get_main_table(TABLE_MAIN_COURSE);
+        $tblExtraFieldValue = Database::get_main_table(TABLE_EXTRA_FIELD_VALUES);
+        $tblLpItem = Database::get_course_table(TABLE_LP_ITEM);
+        $tblLp = Database::get_course_table(TABLE_LP_MAIN);
         $query = "
-        SELECT
-            item_id AS lp_id,
-            REPLACE (s.value, ';', ',') AS users_id
-        FROM
-            $tblExtraFieldValue s
-        INNER JOIN $tblExtraField sf ON (s.field_id = sf.id)
-        WHERE
-            field_id IN (
-                SELECT
-                    id
-                FROM
-                    $tblExtraField
-                WHERE
-                    variable = 'authors'
-            )
-        AND sf.extra_field_type = ".ExtraField::FIELD_TYPE_DATE."
-        AND (s.value != '' OR s.value IS NOT NULL)
-";
+        SELECT DISTINCT
+            lp.name,
+            lpi.title,
+            lp.id as lp_id,
+            lpi.id AS lp_item_id,
+            REPLACE (efv.`value`, ';', ',') AS users_id,
+            c.title AS course_title,
+            c.`code` AS course_code
+        FROM $tblExtraFieldValue AS efv
+        INNER JOIN $tblExtraField AS ef
+        ON (
+            efv.field_id = ef.id AND
+            ef.variable = 'authorlpitem' AND
+            efv.`value` != '' )
+        INNER JOIN $tblLpItem AS lpi
+        ON ( efv.item_id = lpi.iid )
+        INNER JOIN $tblLp AS lp
+        ON ( lpi.lp_id= lp.iid )
```

Comment:
Falta agregar los c_id y corregir el spacing:

`ON (lpi.lp_id= lp.iid AND lpi.c_id = lp.c_id)`

Reply:
Lo mismo para los otros queries